.rubocop.yml

```diff
@@ -50,85 +119,37 @@ Lint/LiteralInCondition:
 Lint/Loop:
   Enabled: true
 
-# do not put space before arguments when they are in parentheses
+# Do not put space before arguments when they are in parentheses
 Lint/ParenthesesAsGroupedExpression:
   Enabled: true
 
 # Do not rescue Exceptions class itself
 Lint/RescueException:
   Enabled: true
 
-# do not shadow local variables in blocks, choose other name
+# Do not shadow local variables in blocks, choose other name
 Lint/ShadowingOuterLocalVariable:
   Enabled: true
 
-# use _ or variable_name to explicitly mark variable as unused
+# Use _ or variable_name to explicitly mark variable as unused
 Lint/UnusedBlockArgument:
   Enabled: true
 
-# use _ or _argument_name to explicitly mark argument as unused
+# Use _ or _argument_name to explicitly mark argument as unused
 Lint/UnusedMethodArgument:
   Enabled: true
 
-# avoid useless assignment
+# Avoid useless assignment
 Lint/UselessAssignment:
   Enabled: true
 
 # Do not use variables in void context
 Lint/Void:
   Enabled: true
 
-# Align the elements of a hash literal if they span more than one line
-Style/AlignHash:
-  Enabled: true
-
-# Align the parameters of a method call if they span more than one line
-Style/AlignParameters:
-  Enabled: true
-
-# Use && instead of and, use || instead of or
-Style/AndOr:
-  Enabled: true
-
-# avoid deep blocks nesting
-Metrics/BlockNesting:
-  Max: 4
-
-Metrics/ClassLength:
-  Max: 300
-  Exclude: 
-    - 'app/models/conference.rb'
-
-# avoid redundunt curly braces when it is obvious that hash is used
-Style/BracesAroundHashParameters:
-  Enabled: true
-
-# Checks that operators have space around them, except for ** which should not have surrounding space.
-Style/SpaceAroundOperators:
-  Enabled: true
-
-Style/CaseEquality:
-  Enabled: true
-
-Style/ClassAndModuleChildren:
-  Enabled: true
-
-Style/StringLiterals:
-  Enabled: true
-
-Style/TrailingBlankLines:
-  Enabled: true
+#################### Rails ###############################
 
-Style/TrailingWhitespace:
+# Preferred guidelines for Rails code
+Rails:
   Enabled: true
-
-AllCops:
-  Include:
-    - '**/Rakefile'
-    - '**/config.ru'
-  Exclude:
-    - 'db/schema.rb'
-    - 'vendor/bundle/**/*'
-    - 'bundle/**/*'
-    - 'config/**/*'
-    - 'bin/*'
+inherit_from: .rubocop_todo.yml
```

Comment:
This line should be on top, the first line of this file. As it is importing a file thst overwrite this one,.